indico/htdocs/js/indico/angular/directives.js

```diff
@@ -468,3 +468,21 @@ ndDirectives.directive('ndInitFocus', function() {
         }, 0);
     };
 });
+
+ndDirectives.directive('ndJqueryDatepicker', function() {
+    return {
+        require: 'ngModel',
+        restrict: 'A',
```

Comment:
I really wonder who the hell thought it's a good idea to use single-char values for this (same with the `@`/`=` in `$scope`)... weird angular people for sure!